zipkin/src/main/java/zipkin/collector/Collector.java

```diff
@@ -74,23 +78,34 @@ public Builder sampler(CollectorSampler sampler) {
       return this;
     }
 
+    /** @see {@link CollectorComponent.Builder#filters(List<SpanFilter>)} */
+    public Builder filters(List<SpanFilter> filters) {
+      logger.info("Total filters loaded: " + filters.size());
```

Comment:
remove this.. as we don't log builder invocations. probably can accomplish the same at higher abstraction ex /autoconfig endpoint